omegatrade-app/frontend/src/app/components/company/update-company/update-company.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="spinner" *ngIf="this.loader">
+    <mat-spinner></mat-spinner>
+</div>
+<h2 mat-dialog-title>{{ (data && data.companyId)?'Edit':'Create' }} Company</h2>
```

Comment:
For edit case, we can show company name like
**Edit Company Searce**

Reply:
the above suggestion has been added.